### DISCUSSION TITLE
TeslaScope Import

### DISCUSSION THREAD

**Discussion by bstadnik1** [teslamate-org/teslamate#1521] [Q&A]
Hi, I had been using TelsaScope to log all my info, but recently found TeslaMate and was wondering if data from TeslaScope could be imported to TeslaMate?

**Comment by eliluong** [comment]
Unfortunately, the only supported format is TeslaFi (https://docs.teslamate.org/docs/import/teslafi). tesla-apiscraper was another popular self-hosted data collection app, and someone wrote a script to convert tesla-apiscraper format to TeslaFi format for import into TeslaMate. So you would probably have to write your own script to import.

**Reply by jannyg** [reply]
Did you create a script to convert the Teslascope export to a compatible format? I only have a few days of data exported from Teslascope so i won't spent too much time on it myself, but it would be nice if someone already wrote sonething 🙂

**Reply by eliluong** [reply]
I did not use teslascope, but tesla-apiscraper.
